s3/replication/manager/tests/system/test_subscribers_resource.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import aiohttp
+import pytest
+
+from fixtures.subscribe import subscriber_record  # noqa: F401;
+
+# Global subscriber id to perform validations across test cases.
+global_valid_subscriber_id = ""
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "test_case_name, expected_http_status",
+    [('valid_payload', 201),
+     ('empty_payload', 500)])
+async def test_post_subscriber(logger, test_config,
+                               subscriber_record,  # noqa: F811;
+                               test_case_name, expected_http_status):
+    """Post subscriber tests."""
+    test_data = {'valid_payload': subscriber_record, 'empty_payload': {}}
+
+    test_payload = test_data[test_case_name]
+
+    async with aiohttp.ClientSession() as session:
+        # Add subscriber
+        async with session.post(test_config['url'] + '/subscribers',
+                                json=test_payload) as response:
+            logger.debug('HTTP Response: Status: {}'.format(response.status))
+            if test_case_name == 'valid_payload':
+                response_body = await response.json()
+                logger.debug('HTTP Response: Body: {}'.format(response_body))
+                global global_valid_subscriber_id
```

Comment:
global is not required may be, as we are not modifying value of 'global_valid_subscriber_id'.

Reply:
We are using it in further tests hence global